spec/factories/organization.rb

```diff
@@ -99,6 +99,11 @@
       end
     end
 
+    after(:create) do |organization, evaluator|
```

Comment:
I'm thinking we should avoid setting all statuses to `selected` because there are many test cases that rely on certain statuses being/not being selected. If Organizations create statuses with default statuses selected, then we can just update the statuses to selected in specs that expect them to be selected.